mitodl/open-discussions
# Fix backpopulate not adding comments

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1787 

#### What's this PR do?
Switches how we create comments to address the link issue plus any potential write conflicts due to race conditions.

#### How should this be manually tested?
Delete a `Comment` record, run `./manage.py backpopulate_posts` and it should be recreated.
